yarn.lock

```diff
@@ -0,0 +1,10034 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.

```

Comment:
Add this file to the .gitignore (and remove it), to avoid having issues with this and package-lock.json.